core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -269,6 +277,12 @@ class WorldScreen(val viewingCiv:CivilizationInfo) : CameraStageBaseScreen() {
         }
     }
 
+    private fun getSpectatorCiv(): CivilizationInfo? {
```

Comment:
I don't think this function is necessary
It seems that this function is only called when fog of war is false
And in that situation, we can just use viewingCiv, since the only time fogOfWar is false is when viewingCiv is spectator, no?

Reply:
You are right fogOfWar could be false only in spectator mode. Thought could some meaningful fun name for clarity. But seems simple update(viewingCiv) would be enough.

Fixed it